src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -27,6 +29,7 @@ internal class OutsideRuntimeClient : IRuntimeClient, IDisposable, IClusterConne
         private Logger callBackDataLogger;
         private ILogger timerLogger;
         private ClientConfiguration config;
+        private IOptions<ClientMessagingOptions> clientMessagingOptions;
```

Comment:
just keep the ClientMessagingOptions, no need for the `IOptions<T>` wrapper, right?